loom/common/platform/platformFile.c

```diff
@@ -99,6 +89,11 @@ int platform_removeFile(const char *path)
     return remove(path);
 }
 
+int platform_moveFile(const char *source, const char *dest)
+{
+    return rename(source, dest);
```

Comment:
I think all platforms implement it, we use `remove` above as just a syscall as well. We'll see what build box says.